README.md

```diff
@@ -834,6 +834,18 @@ In either case:
 
     # good
     names.select { |name| name.start_with?("S") }.map { |name| name.upcase }
+    
+    # bad
```

Comment:
it seems like the example immediately before and immediately after these two are a pair (they have the same behavior); should we make sure to keep them adjacent?